_config.php

```diff
@@ -6,3 +6,11 @@
 $exts = Config::inst()->get('File', 'allowed_extensions');
 $exts[] = 'sspak';
 Config::inst()->update('File', 'allowed_extensions', $exts);
+
+// This will ensure jobs can correctly clean themselves up on any type of failure
+Resque_Event::listen('onFailure', function(Exception $exception, Resque_job $job) {
+	$inst = $job->getInstance();
+	if($inst instanceof DeploynautJobInterface) {
+		$inst->onFailure($exception);
```

Comment:
Nah, I thought it could be used in the future. eg. The instance class may have a logger attached to it which would log the exception message. I can remove if you don't think its worth having.